examples/apps/aogh/aogh.cpp

```diff
@@ -0,0 +1,191 @@
+        OT_API_CALL(otThreadSetExtendedPanId(otrGetInstance(), &kTestNetworkXPanId));
+        OT_API_CALL(otLinkSetPanId(otrGetInstance(), kTestNetworkPanId));
+        OT_API_CALL(otLinkSetChannel(otrGetInstance(), kTestNetworkChannel));
+        OT_API_CALL(otThreadSetMasterKey(otrGetInstance(), &kTestNetworkKey));
+        OT_API_CALL(otIp6SetEnabled(otrGetInstance(), true));
+        OT_API_CALL(otThreadSetEnabled(otrGetInstance(), true));
+        // simply wait for a small while
+        vTaskDelay(pdMS_TO_TICKS(2000));
+    }
+
+    void InitializeLED(void)
+    {
+        mBrightness = 0;
+        for (int i = 0; i < LED_NUM; ++i) {
+            nrf_gpio_cfg_output(LED_PIN[i]);
+            nrf_gpio_pin_write(LED_PIN[i], TURN_OFF);
+        }
+    }
+
+    int AoghServer(void)
+    {
+        int          sockfd = socket(AF_INET6, SOCK_DGRAM, 0);
+        int          ret    = 0;
+        sockaddr_in6 bindAddr;
+        sockaddr     peerAddr;
+        socklen_t    addrLen;
+
+        addrLen = sizeof(peerAddr);
+        bindAddr.sin6_family = AF_INET6;
+        bindAddr.sin6_addr   = IN6ADDR_ANY_INIT;
+        bindAddr.sin6_port   = htons(kAoghPort);
+
+        SuccessOrExit(ret = bind(sockfd, reinterpret_cast<sockaddr *>(&bindAddr), sizeof(bindAddr)));
+
+        while (true)
+        {
+            socklen_t size;
+            VerifyOrExit((size = recvfrom(sockfd, reinterpret_cast<char *>(mBuf), sizeof(mBuf), 0, &peerAddr,
+                                          &addrLen)) >= 0);
+            UdpHandler(sockfd, mBuf, peerAddr, size);
+            taskYIELD();
+        }
+    exit:
+        printf("ERROR: failed to bind socket. sockfd = %d ret = %d\n", sockfd, ret);
+        return ret;
+    }
+
+    void UdpHandler(uint8_t aSockFd, const uint8_t *aData, const sockaddr &aPeerAddr, uint32_t aSize)
+    {
+        uint8_t command = *aData;
+        switch (command) {
+            // respond six bytes as a mock mac address to multicast scan
+            case FIRST_BYTE_OF_MULTICAST:
+                uint8_t macAddress[MAC_ADDRESS_SIZE];
+                EuiToMacAddress(macAddress);
+                sendto(aSockFd, &macAddress, sizeof(macAddress), 0, &aPeerAddr, sizeof(aPeerAddr));
+                return;
+
+            // turn off all LEDs
+            case CMD_OFF:
+                ControlLED(0);
+                break;
+
+            // turn on all LEDs
+            case CMD_ON:
+                ControlLED(100);
+                break;
+
+            case CMD_BRIGHTNESS:
+                ControlLED(*(aData + 1));
+                break;
+
+            default:
+                printf("ERROR: unrecognized packet\n");
+        }
+
+        sendto(aSockFd, &mBrightness, sizeof(mBrightness), 0, &aPeerAddr, sizeof(aPeerAddr));
+    }
+
+    void EuiToMacAddress(uint8_t *aMacAddress)
+    {
+        otExtAddress extAddress;
+        OT_API_CALL(otLinkGetFactoryAssignedIeeeEui64(otrGetInstance(), &extAddress));
```

Comment:
A blank line between declaration and code.
See https://github.com/openthread/openthread/blob/master/STYLE_GUIDE.md